kokoro/gcp_ubuntu_docker/kokoro_build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Fail on any error.
+set -e
+
+# Display commands being run.
+# WARNING: please only enable 'set -x' if necessary for debugging, and be very
+#  careful if you handle credentials (e.g. from Keystore) with 'set -x':
+#  statements like "export VAR=$(cat /tmp/keystore/credentials)" will result in
+#  the credentials being printed in build logs.
+#  Additionally, recursive invocation with credentials as command-line
+#  parameters, will print the full command, with credentials, in the build logs.
+# set -x
+
+# Code under repo is checked out to ${KOKORO_ARTIFACTS_DIR}/github.
+# The final directory name in this path is determined by the scm name specified
+# in the job configuration.
+cd "${KOKORO_ARTIFACTS_DIR}/github/hadoop-migration-assessment-tools"
+pwd
```

Comment:
`pwd` and `ls -l` do not seem relevant.

What's the reason of having separate .sh file to run `bazel build //dist:all`? Is it expected to be used in other places? I would suggest just inlining the command here.

Reply:
Could you elaborate on having a separate `build.sh` file, please?

Reply:
we can have all the build steps in this file and actual kokoro_build will be clean